woodwork/logical_types.py

```diff
@@ -70,8 +71,14 @@ def transform(self, series):
                 raise TypeConversionError(series, new_dtype, type(self))
         return series
 
-    def validate(self, series, return_invalid_values=True):
-        return
+    def validate(self, series, *args, **kwargs):
```

Comment:
That is because there are 2 parameters in the override method of validate in some sub-class of LogicalType, such as EmailAddress, URL etc. We have to pass enough parameters through the root class, otherwise it will throw an error.

Reply:
There are 2 parameters required by validate method of some of sub-class like EmailAddress and URL. As it's used at ww.validate_logical_type to validate for each columns, we have to handle either 1 param like Ordinal, or 2 params like URL. For now the easiest way is in base class. Otherwise we need modify ww.validate_logical_type to decide call validate with 2 params or 1 param.

Reply:
yep, you're right that we need something--either these args or the `return_invalid_values`--but I think that's also the case for the Ordinal column. If I do the following, I get a TypeError `validate() got an unexpected keyword argument 'return_invalid_values'`.

```
from woodwork.logical_types import Ordinal

df = pd.DataFrame({'age':[33,33,44,57], 'latlong':[(1,2), (3,4), (5,6), (8,9)]})
df.ww.init(logical_types={'age':Ordinal(order=[44,33,57]), 'latlong':'Latlong'})
df.ww.validate_logical_types()
```
This didn't show up because the Ordinal and Latlong types aren't being tested in `validate_logical_types`, which we probably want.

Reply:
good catch. So what's your suggestion, to modify ww.validate_logical_types, or put *args, **kwargs into validate function of Ordinal and Latlong?

Reply:
In the long run, we'll want to actually handle `return_invalid_values` in the ordinal and latlong logical types. 

But at this point, let's leave that for their own issue and PR. For now, maybe just add the `return_invalid_values` parameter back into the Ordinal, and Latlong classes (but not the LogicalType.validate one because we don't want to handle it there) and leave TODOs in a comment to handle the logic for determining which individual values. 

I'll make the follow-up issue

Reply:
https://github.com/alteryx/woodwork/issues/1336

Reply:
updated